app/routes/task_routes.py

```diff
@@ -1 +1,104 @@
-from flask import Blueprint
\ No newline at end of file
+from flask import Blueprint, request, abort, make_response, Response
+from app.models.task import Task
+from datetime import datetime
+from app.routes.route_utilities import validate_model
+from ..db import db
+from app.routes.route_utilities import send_slack_message
+
+bp = Blueprint("task_bp", __name__,url_prefix="/tasks")
+
+@bp.post("")
+def create_task():
+    request_body = request.get_json() 
+
+    try:
+        new_task = Task.from_dict(request_body)
+    except:
+        response = {"details": "Invalid data"}
+        abort(make_response(response, 400))
+
+    db.session.add(new_task)
+    db.session.commit()
+
+    return new_task.to_dict(), 201
+
+@bp.get("")
+def get_all_tasks():
+
+    query = db.select(Task) #select records from db Model
+
+    # Check for sorting parameter and apply
+    sorting_param = request.args.get("sort", "asc")
+    
+    if sorting_param == "desc":
+        query = query.order_by(Task.title.desc())
+    else:
+        query = query.order_by(Task.title)
+
+    #query = query.order_by(Task.id)#select records
+    tasks = db.session.scalars(query) #retrieve the records
+    
+    response =[]
+    for task in tasks:
+        response.append(task.to_dict(include_name=False))
```

Comment:
Could this sorting logic be placed inside of our `get_models_with_filters` function?